lib/matplotlib/figure.py

```diff
@@ -448,29 +448,35 @@ def autofmt_xdate(self, bottom=0.2, rotation=30, ha='right'):
         bottom subplot and turn them off on other subplots, as well as
         turn off xlabels.
 
-        *bottom*
+        Parameters
+        ----------
+
+        bottom : scalar
             The bottom of the subplots for :meth:`subplots_adjust`
 
-        *rotation*
+        rotation : angle in degrees
             The rotation of the xtick labels
 
-        *ha*
+        ha : string
             The horizontal alignment of the xticklabels
+
+        which : {None, 'major', 'minor', 'both'}
+            Selects which ticklabels to rotate
         """
         allsubplots = all(hasattr(ax, 'is_last_row') for ax in self.axes)
         if len(self.axes) == 1:
-            for label in self.axes[0].get_xticklabels():
+            for label in self.axes[0].get_xticklabels(False, which):
```

Comment:
You can also pass this in as `get_xticklabels(which=which)` which is less cryptic than the leading `False`.

Which we support _both_ `minor` and `which` is an interesting question...which the answer to is it is my fault: 08d961a176c3376571f63fb3b7ae01e56137cde7 😞 .